Resources/public/mapbender.element.digitizer.js

```diff
@@ -1029,11 +1029,12 @@
                 var printButton = {
                     text: Mapbender.digitizer_translate('feature.print'),
                     click: function () {
-                        var printWidget = $('.mb-element-printclient').data('mapbenderMbPrintClient');
-                        if (printWidget) {
+                        widget.printWidget = widget.printWidget || $('.mb-element-printclient').data('mapbenderMbPrintClient');
```

Comment:
We always should use Mapbender.elementRegistry. Otherwise this will fail if you open the standard PrintClient, close it and then try to open it trough the digitizer.